sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -75,8 +75,9 @@ public final class BridgeInternal {
     private BridgeInternal() {}
 
     @Warning(value = INTERNAL_USE_ONLY_WARNING)
-    public static CosmosDiagnostics createCosmosDiagnostics(DiagnosticsClientContext diagnosticsClientContext) {
-        return new CosmosDiagnostics(diagnosticsClientContext);
+    public static CosmosDiagnostics createCosmosDiagnostics(
```

Comment:
should we add this method to ImplementationHelper instead. This will be showed up as public API change

Reply:
Fixed in next iteration